app/views/applications/show.html.erb

```diff
@@ -0,0 +1,63 @@
+<h1><%= @application.name %></h1>
+
+<h3>Street Address: <%= @application.street_address %></h3>
+<h3>City: <%= @application.city %></h3>
+<h3>State: <%= @application.state %></h3>
+<h3>Zip Code: <%= @application.zip_code %></h3>
+<h3>Reason for applying: <%= @application.description %></h3>
+<h3>Application Status: <%= @application.status %></h3>
+
+<h3>Applying for pets: </h3>
+<ul>
+    <% @application.pets.each do |pet| %>
+        <li><p><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></p></li>
+    <% end %>
+</ul>
+
+<% if @application.pets.present? %>
+    <% if @application.status == "In Progress" %>
+        <%= form_with url: "/applications/#{@application.id}", method: :get, local: true do |f| %>
+            <h3><%= f.label :pet_ownership_description, "Why would you be a good owner to the pet(s)?" %></h3>
+            <%= f.text_area :pet_ownership_description %><br><br>
+            <%= f.submit "Submit Application" %>
+        <% end %>
+    <% end %> 
+<% end %>
+
+<%# <p><%= link_to "Update #{@application.name}", "pets/#{@application.id}/edit" %></p>
+<%# #<p><%= link_to "Delete #{@application.name}", "/pets/#{@application.id}", method: :delete %></p>
+
+<% if @application.status == "In Progress" %>
+    <%= form_with url: "/applications/#{@application.id}", method: :get, local: true do |f| %>
+        <h3><%= f.label :search, "Search for a pet to add:" %></h3>
+        <%= f.text_field :search %>
+        <%= f.submit "Search" %>
+    <% end %>
+
+    <h3>Adoptable Pets:</h3>
+    <% if @pets.nil? %> 
+        <p>No Pets Found </p>
+    <% else %> 
+        <ol>
+            <% @pets.each do |pet| %>
+                <li><%= pet.name %></li>
+                <ul>
+                    <li><%= pet.age %></li>
+                    <li><%= pet.breed %></li>
+                    <li><%= pet.adoptable %></li>
+                </ul>
+                </li>
+                <%= form_with url: "/applications/#{@application.id}", method: :get, local: true do |f| %>
+                    <%= f.hidden_field :added_pet, value: pet.id %>
+                    <%= f.submit "Adopt #{pet.name}" %>
```

Comment:
This isn't quite the functionality we are expecting for the search, but it does achieve the same goal! Instead of listing every adoptable pet, you could opt instead just for a search keyword that you send as a query param, and then use that param to pass to Pet.search